static/js/components/JobDetails.jsx

```diff
@@ -1,13 +1,33 @@
 var AddForm = React.createClass({
 
   getInitialState: function() {
-    return {clicked: false, errored: false};
+    return {clicked: false, errored: false, JSON_workload_checked: false};
+  },
+
+  raiseJSONWorkloadWarning: function(workload) {
+    if (workload.length === 0 || workload[0] != "{") {
```

Comment:
I can't think of any case off hand that would wold warrant no trimming both leading and trailing spaces.